Cargo.toml

```diff
@@ -21,11 +21,11 @@ bitcoin_hashes = { version = "0.3", features = ["serde"] }
 btc-transaction-utils = "0.5"
 byteorder = "1.3"
 derive_more = "0.15"
-exonum = "0.11.0"
-exonum-merkledb = { git = "https://github.com/exonum/exonum.git", rev = "33e45eac5c9d710a100ce80b56d2d31a5b1f6dfc" }
+exonum = "0.12.0"
+exonum-merkledb = "0.12.0"
```

Comment:
Again, you can use reexported merkledb from `exonum` crate (to avoid possible versions conflicts in the future).